roll.py

```diff
@@ -30,6 +30,10 @@
     "l1",
     help="spins up a local L1 node with the rollup contracts deployed on it")
 
+subparsers.add_parser(
+    "l2",
+    help="spins up a local op-geth node")
```

Comment:
I know this is the naming scheme for the Optimism repo but I suggest we switch to something more descriptive like "l2-execution". Then we can use "l2" as the target to deploy every part of an L2 (execution, proposer, batcher).

Reply:
I think l2-execution is probably a little bit more clear, but both are really fine by me.

Reply:
I'll take care of this later!